package.json

```diff
@@ -57,6 +57,7 @@
     "@typescript-eslint/parser": "^5.19.0",
     "@vitejs/plugin-vue": "^2.3.1",
     "@vitest/ui": "^0.9.3",
+    "@vue/compiler-sfc": "^3.2.33",
```

Comment:
I think this doesn't need to be installed separately. Comes with vue.

Reply:
Yep true :)